### PR TITLE
add integration tests for user and employee

### LINKED ISSUE #59: Add integration tests

### PR DESCRIPTION
**Pull request by samflattery**

### CODE REVIEW

File: frontend/src/App.test.tsx [hunk 1]
```diff
@@ -0,0 +1,116 @@
...
+  const quantities = screen.getAllByLabelText(/quantity input/i);
+
+  // Map donut i to how many of i are being purchased
+  const quantityMap =
+    { 0 : 2,
+      1 : 1,
+      2 : 0
+    };
+
+  for (var i in quantityMap) {
```
Comment (Alex7Li) [inline frontend/src/App.test.tsx:43]: Better to never use var, and use let

File: frontend/src/App.test.tsx [hunk 2]
```diff
@@ -0,0 +1,116 @@
...
+  const username = await screen.findByPlaceholderText(/username/i);
+  const password = screen.getByPlaceholderText(/password/i);
+
+  user.type(username, "admin");
+  user.type(password, "test");
+
+  const submitButton = screen.getByText(/submit/i);
+  user.click(submitButton);
+
+  await screen.findByText(/success/i);
```
Comment (Alex7Li) [inline frontend/src/App.test.tsx:103]: Maybe I don't understand this, is this waiting until the screen displays the word success? If so, does it wait indefinitely or do these tests timeout? We want them to timeout and fail eventually

**Review by Alex7Li — APPROVED** [review]
Great, thanks so much for getting this done. I love the work you did with these tests!

File: frontend/src/App.test.tsx (hunk 1 above)
Comment (samflattery) [reply frontend/src/App.test.tsx:43]: Fixed

**Pull request merged**